examples/compiled/selection_type_interval_invert.vg.json

```diff
@@ -298,7 +298,7 @@
         "update": {
           "fill": [
             {
-              "test": "!(length(data(\"pts_store\"))) || (!(vlSelectionTest(\"pts_store\", datum)))",
+              "test": "!(!length(data(\"pts_store\")) || vlSelectionTest(\"pts_store\", datum))",
```

Comment:
This example is now different from before. Is that intended? 

The docs say "Notice, all marks are initially colored grey. This is because empty selections are treated as containing all data values." so I think either the example or the docs need to change.